modules/querying/pages/data-types.adoc

```diff
@@ -451,18 +451,42 @@ include::appendix:example$investment_net/datatypes_udt_results.json[]
 
 == `FILE` Object
 
-A `FILE` object is a sequential data storage object, associated with a text file on the local machine.
+A `FILE` object is a sequential data storage object, associated with a text file on the local machine or on S3 bucket.
 
 [NOTE]
 ====
 When referring to a `FILE` object, we always capitalize the word `FILE` to distinguish it from ordinary files.
 ====
 
+=== Local disk file
 When a `FILE` object is declared, associated with a particular text file, any existing content in the text file will be erased.
 During the execution of the query, content written to the `FILE` will be appended to the `FILE`.
 When the query where the `FILE` was declared finishes running, the `FILE` contents are saved to the text file.
 
-A `FILE` object can be passed as a parameter to another query. When a query receives a `FILE` object as a parameter, it can append data to that `FILE`, as can every other query which receives this `FILE` object as a parameter.
+=== S3 object
```

Comment:
In the interest of just publishing something, I guess I will accept this. But I don't think this configuration information belongs here. This "data types" page is intended to explain the characteristics of each data type, not how to use them.